### ISSUE TITLE
"current_tvb_neu[1]" row missing in sim.py

### ISSUE THREAD

**Issue by madhushalinib** [NIDCD/lsnm_in_python#3]
I was running the software with the following parameters:

Model description file: visual_model/model.txt
Weights list file: visual_model/subject_tvb/tutorial/weightslist.txt
Simulation script file: visual_model/subject_tvb/tutorial/script_DMS_198_seconds.py
Use TVB connectome: visual_model/subject_tvb/tutorial/lsnm_tvb_link.txt

To which the following error was given:

INFO log level set to INFO
WARNING File 'average orientations' not found in ZIP.
WARNING File 'cortical' not found in ZIP.
WARNING File 'hemispheres' not found in ZIP. 
INFO white noise configured with dt=5
Traceback (most recent call last):
   file "/.././simulation/s1n.py", line 855, in run
     current tvb_abs_syn[o][tvb node] += w_ee * current_tvb_neu[o][tvb node] + w_ie * current_tvb_neu[1]t
**IndexError: index 1 is out of bounds for axis 0 with size 1**

This line is located on line 840 of sim.py in the github page. The same issue exists on lines 840, 844, 851, and 852.

The row  "current_tvb_neu[1]" does not exist. Why is this row missing and how can I fix it?

Any suggestions?
Thank you for your time.

**Comment by ulloaa** [comment]
Hello, @madhushalinib,

Thanks for your feedback.

Those files were external to the code and were not included in the github repository at the time due to their size. I will try to use a new github feature to host large data files to upload it to the repo.

Kind regards,

**Comment by ulloaa** [comment]
Hello, @madhushalinib, could you please verify that you are using Python 2.7?

**Comment by madhushalinib** [comment]
Thank you! Yes, I am using Python 2.7.

**Comment by ulloaa** [comment]
OK, thanks. I am trying to reproduce the error on my end.

**Comment by ulloaa** [comment]
I have another question: do you have a conda environment that you built to run lsnm or no conda environments? Also, on what platform are you running it (Mac, PC, Linux)? I am thinking it could be a syntax issue using recent packages since I have not updated the code in so long. Thanks again,

**Comment by madhushalinib** [comment]
Yes, I have built a conda environment and have been using a Linux platform.

**Comment by ulloaa** [comment]
Would you mind sending the output of “conda list” and “pip list” from within your environment? That way I can reproduce. Thanks!

**Comment by madhushalinib** [comment]
I have attached the outputs below. Thank you for your time!

Pip list:
![pip_list](https://user-images.githubusercontent.com/60161206/213068888-c527c1b6-6ae2-41c9-a236-dc5e03a8a7cc.png)
Conda list:
![conda_list1](https://user-images.githubusercontent.com/60161206/213069201-488b65a6-2fb8-4a8a-a1fb-89e01eaa804f.png)
![conda_list2](https://user-images.githubusercontent.com/60161206/213069211-c8cf28cf-763b-4ecb-964e-c9b4a29f9ebe.png)

**Comment by madhushalinib** [comment]
If you are available, do you think it would be possible for us to have a zoom meeting later? I am a junior in high school and am using LSNM to perform an experiment as part of my science fair project. I have a couple of questions regarding LSNM and was wondering if we could discuss this and any suggestions you might have pertaining to my science fair project if you have the time. Could you please contact me at ? I greatly appreciate it!
